### PR TITLE
[MNG-6581] - Update to latest Modello plugin version

### PR DESCRIPTION
**Pull request by oehme**
Follow-up to https://github.com/codehaus-plexus/modello/pull/31

Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MNG) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[MNG-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MNG-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [x] You have run the [Core IT][core-its] successfully.

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

[core-its]: https://maven.apache.org/core-its/core-it-suite/

### CODE REVIEW

**Comment by olamy** [comment]
we might need a Jira here :)

**Comment by oehme** [comment]
Done :)

**Comment by olamy** [comment]
thanks!
pushed in branch MNG-6581 let's see what CI says

File: pom.xml [hunk 1]
```diff
@@ -62,7 +62,7 @@ under the License.
     <wagonVersion>3.3.2</wagonVersion>
     <securityDispatcherVersion>1.4</securityDispatcherVersion>
     <cipherVersion>1.7</cipherVersion>
-    <modelloVersion>1.10.0</modelloVersion>
```
Comment (michael-o) [inline pom.xml]: We will have to wait until 1.10.1 is released, we don't use snapshots permanently.

Comment (olamy) [reply pom.xml]: as said it's only for CI testing. It's green so I can release modello

**Comment by michael-o** [comment]
@rfscholte @khmarbaise Can someone push a new Modello release?

**Comment by olamy** [comment]
oops sorry I will do that now

**Comment by olamy** [comment]
@oehme modello 1.11 released please update the pr.

**Comment by michael-o** [comment]
ITs are running...

**Pull request closed without merging**